Sofya_Huliuk_Application/Program.cs

```diff
@@ -294,8 +294,147 @@ public void Task_9()
         uint d = (uint)b;
     }
 
+    public void Task_10()
+    {
+
+    }
+
 
 
 
 
 }
+
+class Homework_3
+{
+    public void Task_1()
+    {
+        while (true)
```

Comment:
Is it really need to use loop here? you don't have a loop exit condition

Reply:
the purpose of the loop here is to test multiple input values instead of just one with a single run of the console application